pkg/util/conditions/merge.go

```diff
@@ -173,6 +174,7 @@ type conditionGroup struct {
 
 // mergePriority provides a priority value for the status and severity tuple that identifies this
 // condition group. The mergePriority value allows an easier sorting of conditions groups.
+// nolint: exhaustive
```

Comment:
Can we keep the exhaustive check here (remove this nolint comment), and add the following to the line 188?

```go
case v1alpha1.ConditionSeverityNone:
	return 10
```